python/pyspark/sql/streaming.py

```diff
@@ -1464,6 +1491,76 @@ def start(self, path=None, format=None, outputMode=None, partitionBy=None, query
         else:
             return self._sq(self._jwrite.start(path))
 
+    def toTable(self, tableName, format=None, outputMode=None, partitionBy=None, queryName=None,
+                **options):
+        r"""
+        Streams the contents of the :class:`DataFrame` to the output table.
+
+        A new table will be created if the table not exists. The returned [[StreamingQuery]]
+        object can be used to interact with the stream.
+
+        .. versionadded:: 3.2.0
```

Comment:
Ideally it'd be nice to ship the change in 3.1.0 so that the API is available for both at the same time. I just safely set this to 3.2.0 to see the voices on when to add.

Reply:
I am okay if other people are fine.

Reply:
I think it is okay as this only expose the APIs to PySpark.